src/main/java/com/jcabi/log/DullyFormatted.java

```diff
@@ -57,11 +57,8 @@ public DullyFormatted(final String bas) {
      */
     @Override
     public String format() {
-        return this.basic.replace(
-            new ControlSequenceIndicatorFormatted("%s?m").format(),
-            ""
-        ).replace(
-            new ControlSequenceIndicatorFormatted("%sm").format(),
+        return this.basic.replaceAll(
+            new ControlSequenceIndicatorFormatted("%s([0-9]*|\\?)m").format(),
             ""
```

Comment:
``%s([0-9]*|\\?)m`` will eventually be formatted to the regex ``\u001b\\[([0-9]*|\\?)m`` which catches everything we need: e.g. ``\u001b[23m``, ``\u001b[?m`` or ``\u001b[m``.